webapp/apps/taxbrain/models.py

```diff
@@ -294,6 +294,16 @@ class TaxSaveInputs(models.Model):
     ID_c_3 = CommaSeparatedField(default=None, blank=True, null=True)
     ID_c_4 = CommaSeparatedField(default=None, blank=True, null=True)
     ID_c_cpi = models.NullBooleanField(default=None, blank=True, null=True)
+    # uncomment once we incorporate new variables
```

Comment:
Is this ready to be uncommented?

Reply:
I was working with a local version of Tax-Calculator, which had these new parameters, that I thought was equivalent to 0.10.1.  So, I made some changes to the webapp to accommodate these new parameters.  Then I realized that they haven't been included in a `taxcalc` package yet.  I figured that they would be added eventually so I just commented the code instead of deleting it.

Reply:
So these models work with 0.10.2 and not 0.10.1?

Reply:
No, this change wasn't included in 0.10.2 either.

Reply:
Ahh. In that case it might be good to remove them. It might be a while before the app uses that latest version.

Reply:
Ok sounds good.